packages/validator/src/slashingProtection/utils.ts

```diff
@@ -28,7 +28,7 @@ export function numToString(num: number): string {
 }
 
 export function minEpoch(epochs: Epoch[]): Epoch | null {
-  return epochs.length > 0 ? Math.min(...epochs) : null;
+  return epochs.length > 0 ? epochs.reduce((minEpoch, epoch) => (minEpoch < epoch ? minEpoch : epoch)) : null;
```

Comment:
alternative implementation

```ts
export function minEpoch(epochs: Epoch[]): Epoch | null {
  if (epochs.length === 0) {
    return null;
  }

  let minEpoch = epochs[0];
  for (let i = 1; i < epochs.length; i++) {
    if (epochs[i] < minEpoch) {
      minEpoch = epochs[i];
    }
  }

  return minEpoch;
}
```

but there is no noticeable difference in performance